Core/RadioHandler.cpp

```diff
@@ -344,8 +344,8 @@ void RadioHandlerClass::CaculateStats()
 	BytesXferred = 0;
 	SamplesXIF = 0;
 
-	uint16_t maxlen = 64;
-	uint8_t  debdata[64];
+	uint16_t maxlen = 100;
```

Comment:
suggest a constant value in Interface.h

Reply:
I placed define MAXLEN_D_USB (100)  in Interface.h  Thanks !